src/script/standard.cpp

```diff
@@ -221,6 +221,7 @@ bool ExtractDestination(const CScript& scriptPubKey, CTxDestination& addressRet)
     return false;
 }
 
+// TODO: from v22 ("addresses" and "reqSigs" deprecated) "ExtractDestinations" should be removed
```

Comment:
This is incorrect. Current master will become v22. This functionality can be removed in v23.